test/flycheck-test.el

```diff
@@ -4225,7 +4236,7 @@ Why not:
      '(5 7 error "unexpected token tCONSTANT" :checker ruby-rubylint))))
 
 (flycheck-ert-def-checker-test ruby ruby syntax-error
-  (let ((flycheck-disabled-checkers '(ruby-rubocop ruby-reek ruby-rubylint)))
```

Comment:
Why did you re-enable the checker in the tests?  I think it's the reason that `make integ` tests fail on the CI.